src/model.py

```diff
@@ -284,6 +293,10 @@ def prettyprint(self) -> str:
             result.append("Drop " + drop_filename)
         for email in self.emails:
             result.append("Email " + email)
+        for ransom_message in self.ransom_messages:
+            result.append("Ransom message: " + ransom_message)
```

Comment:
`RansomMessage`

Reply:
(no `:` to be consistent with other entries, and PascalCase so that descriptions won't be longer than one word